sunpy/map/sources/gong.py

```diff
@@ -61,3 +71,56 @@ def spatial_units(self):
     @property
     def observer_coordinate(self):
         return get_earth(self.date)
+
+
+class GONGHAlphaMap(GenericMap):
```

Comment:
Not sure how I feel about the HAlpha, could it be Halpha? .

Reply:
I don't have strong opinion either way, so happy to change.